index.d.ts

```diff
@@ -20,3 +20,8 @@ declare module '*.svg?raw' {
   const src: string
   export default src
 }
+
+declare module '*.svg?skipsvgo' {
+  const src: string
```

Comment:
I think the type of `src` should be `FunctionalComponent<SVGAttributes>` instead of `string`

Reply:
fixed in 0c7e9dd9c05cea9ae4adbe710ee46a7f49380dcf